portfolio/src/main/webapp/index.html

```diff
@@ -53,16 +53,14 @@ <h2 align="center">Let's keep in touch!</h2>
             <a href="https://www.linkedin.com/in/tengkuhanashafikah/" button class="button button1">LinkedIn</a>
             <a href="https://www.instagram.com/tghanashafikah/" button class="button button1">Instagram</a>
             <button class="button button1" onclick="showImage()">Nothing to see here...</button>
-            <br>
-            <br>
+            <br><br>
 
-            <!–– Test Servlet: Say Hello ––>
-            <p>If you're Hana, please click following button to have the website say hello to you 
-               (because you have no friends ahaha 😛). If you're not Hana then that's fine too.</p>
-            <button class="button button1" onclick="getGreeting()">Say Hello</button>
-            <br>
-            <br>
-            <div id="greeting-container"></div>
+        <!–– Test Servlet: Hard-Coded Comments ––>
+            <p>Below are some comments I wrote to myself (under various names to make
+            the comment section look less sad). Feel free to take a look at them.</p>
+            <button class="button button1" onclick="getComments()">Show Comments</button>
+            <br><br>
+            <div id="comments-container"></div>
```

Comment:
Since you're creating a list here, I would suggest adding an empty list here instead and referencing that in the JavaScript

```suggestion
            <div id="comments-container">
               <ul id="comments-list"/>
            </div>
```

And then in the JS using `document.getElementById('comments-list')` instead.